integration-test.sh

```diff
@@ -134,7 +135,46 @@ function set_up {
         sleep 1
     done ;
     info "ES metrics store is up and running."
-    popd
+    popd > /dev/null
+}
+
+function set_up_proxy_server {
+    # we want to see output to stderr for diagnosing problems
+    if docker ps > /dev/null; then
+        info "Docker is available. Proxy-related tests will be run"
+        # Portably create a temporary config directory for Squid on Linux or MacOS
+        local config_dir=$(mktemp -d 2>/dev/null || mktemp -d -t 'tmp_squid_cfg')
```

Comment:
For Docker on MacOS this requires to (manually) add `/var/folders` to the list of allowed paths (otherwise Docker will complain that it can't bind-mount the directory). `/var/folders` is actually symlinked to `/private/var/folders` and `/private` is already configured (recursively) for Docker but not `/var/folders` is not. I think we should add this to our developer docs. Wdyt?

Reply:
As discussed via another channel we'll instead use an already existing directory tree that we use for integration tests anyway. I'll push a new commit addressing this.